recipes/sdl/all/conanfile.py

```diff
@@ -138,6 +141,8 @@ def requirements(self):
                 self.requires("egl/system")
             if self.options.directfb:
                 raise ConanInvalidConfiguration("Package for 'directfb' is not available (yet)")
+            if self.options.get_safe("libunwind", False):
```

Comment:
```suggestion
            if self.options.libunwind:
```

if linux, this option is available, not need for get_safe here.

Reply:
Good to know! Thank you!